pkgs/apps/openmolcas/v21.02.nix

```diff
@@ -40,9 +49,22 @@ in stdenv.mkDerivation {
     chmod -R u+w External/
   '';
 
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace 'set (libpath "''${MKLROOT}/lib/intel64")' 'set (libpath "''${MKLROOT}/lib")' \
+      --replace 'find_library (LIBMKL_BLACS NAMES "mkl_blacs_ilp64"' 'find_library (LIBMKL_BLACS NAMES "mkl_blacs_intelmpi_ilp64"'
```

Comment:
This requires MPICH/MVAPICH/IntelMPI? There is also a `mkl_blacs_openmpi_ilp64`. Does this work properly?

Reply:
Luckily it does not :slightly_smiling_face: Those are [these lines here](https://gitlab.com/Molcas/OpenMolcas/-/blob/master/CMakeLists.txt#L1238). OpenMPI should already work correctly without the second `--replace` line (If we decide to add MPI support to Molcas later (we can port it from my overlay))

Reply:
Since the substitution here is static it would be better to provide it as patch. `substituteInPlace` does not fail when the change becomes obsolete but a patch does. Then we know that it can be removed (or needs to be adapted).